src/Listener/ToolbarListener.php

```diff
@@ -147,7 +147,7 @@ protected function injectToolbar(ProfilerEvent $event)
      * @return array
      * @throws InvalidOptionException
      */
-    protected function renderEntries(ProfilerEvent $event)
+    protected function renderEntries(ProfilerEvent $event): array
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!